packages/cli/src/lib/commands/migrate-command.ts

```diff
@@ -1,5 +1,7 @@
 import { inject, injectable, injectFromBase } from 'inversify';
 import { Command } from 'commander';
+import fg from 'fast-glob';
+import pLimit from 'p-limit';
```

Comment:
concurrency should be abstracted by a ConcurencyService and this service should be injected in the Usecase